src/control/server/external_mocks.go

```diff
@@ -72,46 +74,52 @@ func (m *mockExt) createEmpty(path string, size int64) error {
 func (m *mockExt) mount(
 	dev string, mount string, typ string, flags uintptr, opts string) error {
 
-	commands = append(
-		commands,
-		fmt.Sprintf("mount %s %s %s %s", dev, mount, typ, opts))
+	op := fmt.Sprintf(msgMount, dev, mount, typ, fmt.Sprint(flags), opts)
+
+	m.history = append(m.history, op)
```

Comment:
using the history both as an informative source to report to user on request (to be implemented) and also for testing to verify expected commands were carried out. yes it is just reporting which functions of the mock get called but seeing as they adhere to an interface it will be the same list of methods.

Reply:
I think it makes a lot of sense in the production code, but perhaps there is a simpler way to verify the mock functions were called in tests. I often use private booleans for that purpose.

Reply:
okay if I bump this suggestion to a future patch?

Reply:
Sure